apps/voting/app/src/app-state-reducer.js

```diff
@@ -26,6 +32,8 @@ function appStateReducer(state) {
       tokenDecimals: tokenDecimalsNum,
     },
 
+    connectedAccountVotes: connectedAccountVotes || {},
```

Comment:
This is to ensure there's an object in https://github.com/aragon/aragon-apps/blob/4afc2ad92c5633eeb7545f716c46703ff42193ab/apps/voting/app/src/vote-hooks.js#L29-L32